localstack/services/events/target.py

```diff
@@ -147,6 +151,11 @@ def send_event(self, event):
             ]
         )
 
+    def _validate_input(self, target: Target):
```

Comment:
This `RoleArn` validation looks redundant. Consider moving it to the `TargetService` base class in a follow-up.

Reply:
it is only required for specific services, this is why it is not in the base class.